ClientApp/viking-vault/src/components/AccountPage.tsx

```diff
@@ -1,37 +1,63 @@
 import React from 'react'
 import './styles.css'
-import account from './images/card.png'
+import inexistentCard from './images/card.png'
+import genericCard from './images/GENERICcard-01.png'
 import {constants} from "../ConstantVariables";
 import { Redirect } from 'react-router-dom';
 import { TransactionList } from '../UserDashboard/TransactionList';
 
-const url = constants.baseUrl+"Accounts";
+const url = constants.baseUrl+"Accounts/";
 
 interface IAccountState{
-    balance: number
-    card: any
-    transactions: []
+    firstName: string
+    lastName: string
+    cardNumber: string
+    expirationDate: string
+    ronBalance: number
```

Comment:
It would make more sense to group the balances in an object since they have multiple properties which are related to each other.

`interface IAccountBalance { eurBalance: number; usdBalance: number }.. blah blah`